### ISSUE TITLE
can't login with supabase 0.15.6+

### ISSUE THREAD

**Issue by brettatoms** [supabase/cli#104]
# Bug report

## Describe the bug

Requests to `http://localhost:54321/auth/v1/token?grant_type=password` return `503
Service Temporarily Unavailable` starting with release 0.15.6 

When checking out sha 5a3db53 `supabase.auth.signIn()` works as expected.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. git clone git@github.com:supabase/cli.git cli
2. git checkout 3a3f20b
3. go build
4. cd <supabase project dir> # mine was created with supabase cli ~0.15.0
5. <supabase cli dir>/cli start
6. Call `supabase.auth.signIn({ email, password })`

## Expected behavior

Authorization succeeds. 

## System information

- OS: macOS
- Browser: Firefox (if applies) [e.g. chrome, safari]
- Version of supabase-js: 1.28.5
- Version of Node.js: v14.16.0

**Comment by brettatoms** [comment]
```
❯ git bisect log  
# bad: [3a3f20bfa808bd13f8ff89c6e0c434ec3b6b1847] fix: etc. etc.
# good: [5a3db5398652b9203df68bdf0108fdda2cff1e76] fix: Apply extensions.sql before migrations
git bisect start '3a3f20bfa808bd13f8ff89c6e0c434ec3b6b1847' '5a3db53'
# bad: [c44acfef6a906ae550760325d8fe91bec63045a4] fix: --host localhost
git bisect bad c44acfef6a906ae550760325d8fe91bec63045a4
# good: [5659834bc3bf49de6d78cd8e83922bcc0ccf1850] chore: more ✨💅✨
git bisect good 5659834bc3bf49de6d78cd8e83922bcc0ccf1850
# bad: [ea41e841346a575d1fce760e8a8b5196bb3c3564] fix: update images
git bisect bad ea41e841346a575d1fce760e8a8b5196bb3c3564
# good: [90a8eae9aa8526bc5e8e99c2743dc5ae764803e3] chore: show `Dumping branches...` when stopping
git bisect good 90a8eae9aa8526bc5e8e99c2743dc5ae764803e3
# first bad commit: [ea41e841346a575d1fce760e8a8b5196bb3c3564] fix: update images
```

**Comment by brettatoms** [comment]
It works after doing `supabase db reset` and creating a new user.

**Comment by soedirgo** [comment]
Hmm, couldn't quite nail down what the issue is, but there was a problem when using the Postgres 14 image, though that should've been resolved in 0.15.6. Will keep this open for a while in case other people are having similar issues.

**Comment by soedirgo** [comment]
Looks to be a heisenbug, so closing for now.
